src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -570,6 +573,12 @@ export class DBTCoreProjectIntegration
         }
       }
     }
+    if (!this.hasManifest) {
+      const initErrorMessage =
+        "The dbt Project is still initializing, please try again later.";
+      window.showErrorMessage(initErrorMessage);
```

Comment:
It's not logic to both throw and show an error message. the error message must be handled somewhere upstream already.